cmd/podsync/updater.go

```diff
@@ -182,6 +182,7 @@ func (u *Updater) downloadEpisodes(ctx context.Context, feedConfig *config.Feed)
 			// We still need to generate XML, so just stop sending download requests and
 			// retry next time
 			if err == ytdl.ErrTooManyRequests {
+				logger.Info("Server responded with a 'Too Many Requests' error")
```

Comment:
I think it should even `Warn` level.
Also could you please make it lower case?

Reply:
Done.